cishouseholds/pipeline/validation_schema.py

```diff
@@ -376,6 +376,19 @@
         "ctry12": string_dict,
         "ctry_name12": string_dict,
         "tranche": integer_dict,
```

Comment:
This line is duplicated below. Please ensure that the dict here matches the list in the spec exactly and try to run the stage with the indicated dummy file to ensure it matches